app/src/main/java/org/p2p/wallet/bridge/api/request/SendEthereumBundleRpcRequest.kt

```diff
@@ -7,7 +7,7 @@ import org.p2p.wallet.bridge.api.response.BridgeBundleResponse
 data class SendEthereumBundleRpcRequest(
     @Transient val bundleRequest: BridgeBundleResponse,
 ) : JsonRpc<Map<String, Any>, Unit>(
-    method = "send_ethereum_bundle", // "simulate_ethereum_bundle",
+    method = "simulate_ethereum_bundle", // "simulate_ethereum_bundle",
```

Comment:
send_ethereum_bundle?

Reply:
for now it's in testing and I use simulating